### ISSUE TITLE
new release since version 1.4.5 ?

### ISSUE THREAD

**Issue by bmx666** [rlabbe/filterpy#277]
Last release 1.4.5 was in 11 Oct 2018. Any news or updates?

**Comment by sciafri** [comment]
Hey @rlabbe there's quite a few issue tickets without response; a couple of which are wondering about a new release based off of the latest commits and I'm also interested in that as well. Obviously this isn't your job and I'm sure you're busy but I figured might catch you not at work with the upcoming holidays. Either way, hope all is well.

Also, might I suggest putting a github "Sponsor" button on the repo or similar, e.g. paetron, somewhere? Certainly this repo and the book would get at least a couple people to donate, I know I would. Finally just wanted to say that I'm getting familiar with this package and I'm planning to make some contributions to the package, e.g. unit testing which I hope will help. Thanks for making this package and the book.

**Comment by catid** [comment]
Yeah it sucks that the pip package is from 5 years ago and has a ton of bugs.  Was using that for a prototype and only realized it was out of date while writing unit tests for the C++ version and saw the calculations were different.

**Comment by kopytjuk** [comment]
@rlabbe your library is amazing and used across academia and industry (due to it simplicity and the explanations), it would be great if you could assign some "successor" maintainers which could review/merge PRs or deal with issues ...

**Comment by AlexAtCCRI** [comment]
@rlabbe  a new release would be greatly appreciated!
